pkgs/top-level/aliases.nix

```diff
@@ -610,6 +610,7 @@ mapAliases ({
   imagemagick7 = imagemagick; # Added 2021-02-22
   imagemagick7_light = imagemagick_light; # Added 2021-02-22
   imlib = throw "imlib has been dropped due to the lack of maintenance from upstream since 2004"; # Added 2023-01-04
+  immich-cli = throw "the legacy immich cli has been dropped in favour of the new 'immich.cli'"; # Added 2024-07-06
```

Comment:
Why not
```suggestion
  immich-cli = immich.cli; # Added 2024-07-06
```
?

Reply:
I asked for this since it's a breaking change.

Reply:
I see. The old immich-cli should at least be removed in a separate commit.